### PR TITLE
Add linting + update dependencies

### LINKED ISSUE #43: Use same lint rules as yt-channel-info
Following the same code style in FreeTube modules will make maintaining a lot easier

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
closes #43

### CODE REVIEW

**Comment by ChunkyProgrammer** [comment]
will open a pr to switch to named object parameters once this is merged

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — DISMISSED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    'standard',
+  ],
+
+  rules: {
+    'space-before-function-paren': 0,
+    'comma-dangle': 0,
+    'no-console': 0,
+    'no-unused-vars': 1,
+    'no-undef': 1,
+    'object-shorthand': 0
```
Comment (PikachuEXE) [inline .eslintrc.js:37]: What do 0 & 1 mean...

File: src/Youtube-Scraper.js [hunk 2]
```diff
@@ -67,11 +67,10 @@ class CommentScraper {
...
     }
 
-    let total = null;
+    let total = null
     if (!continuation) {
-        const headerElem = commentPageResponse.data.onResponseReceivedEndpoints[0].reloadContinuationItemsCommand.continuationItems[0]
-        if ('commentsHeaderRenderer' in headerElem)
-        total = Number(headerElem?.commentsHeaderRenderer?.countText?.runs?.[0]?.text?.replace(',', '')) ?? null
+      const headerElem = commentPageResponse.data.onResponseReceivedEndpoints[0].reloadContinuationItemsCommand.continuationItems[0]
+      if ('commentsHeaderRenderer' in headerElem) { total = Number(headerElem?.commentsHeaderRenderer?.countText?.runs?.[0]?.text?.replace(',', '')) ?? null }
```
Comment (PikachuEXE) [inline src/Youtube-Scraper.js:73]: Why isn't this considered as a long line?
```suggestion
      if ('commentsHeaderRenderer' in headerElem) { 
        total = Number(headerElem?.commentsHeaderRenderer?.countText?.runs?.[0]?.text?.replace(',', '')) ?? null
      }
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (PikachuEXE) [inline .eslintrc.js:37]:
```suggestion
    'space-before-function-paren': 'off',
    'comma-dangle': 'off',
    'no-console': 'off',
    'no-unused-vars': 'warn',
    'no-undef': 'warn',
    'object-shorthand': 'off'
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]
https://eslint.org/docs/latest/user-guide/configuring/rules#configuring-rules

No point making it harder to read later

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Review by absidue — APPROVED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE your health is low, we got to finish this battle fast. Use your ultimate thunder shock approval move now!!!

**Review by PikachuEXE — APPROVED** [review]
It's 9pm in my time zone man = =

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Oops sorry about that its 4pm in mine

**Pull request merged**